store/options.go

```diff
@@ -15,6 +16,9 @@ type Options struct {
 
 	// Tags allows to specify associated tags to the current value
 	Tags []string
+
+	// Ctx pass context for control timeout for all operations
+	Ctx context.Context
```

Comment:
Thanks for fixing this.

I will rework this part later as we should be able to pass the context directly when calling methods from cache (https://github.com/eko/gocache/blob/master/cache/cache.go#L43) package then propagated to codec & stores.

Reply:
get it!